server/id/id_impl.go

```diff
@@ -59,11 +61,11 @@ func (alloc *AllocatorImpl) rebaseLocked(ctx context.Context) error {
 		return errors.Wrapf(err, "get end id failed, key:%v", alloc.key)
 	}
 
-	if currEnd != "" {
-		return alloc.doRebase(ctx, decodeID(currEnd))
+	if currEnd == "" {
```

Comment:
As far as I know, the `currEndID` should be thought as 0 if the key doesn't exist, and why should error be thrown here?